posts/2022-05-03-microstream-cdi-ext.adoc

```diff
@@ -0,0 +1,265 @@
+---
+layout: post
+title: "Easy and Fast Data Access in Open Liberty with MicroStream CDI Extension"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/rdebusscher
```

Comment:
You can also add Otavio here as well. Do you want to display your github repo?

Reply:
Is this a link to the GitHub repo or used for displaying an author picture?

Reply:
@rdebusscher I think Emily was referring to the following line (7), see my suggestion